oak_containers_syslogd/src/main.rs

```diff
@@ -19,25 +19,51 @@
 mod log_relay;
 mod systemd_journal;
 
+use std::sync::Arc;
+
```

Comment:
super nit: for consistency avoid a space between different blocks of `use` statements.